src/psyclone/tests/line_length_test.py

```diff
@@ -330,6 +330,32 @@ def test_edge_conditions_comments():
     assert output_string == expected_output
 
 
+def test_lone_lines_allocate(fortran_reader, fortran_writer):
```

Comment:
You could keep this test but I would suggest testing the method `find_break_point` directly - you can then give it a number of different strings by e.g. parameterising the test (pytest parametrize).

Reply:
Done